torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -1067,6 +1086,22 @@ RegisterOperators reg(
            push(stack, a.is_cuda());
          },
          aliasAnalysisFromSchema()),
+     OperatorGenerator(
+         TORCH_SELECTIVE_SCHEMA("prim::real(Tensor a) -> Tensor"),
+         [](Stack* stack) {
+           at::Tensor a;
+           pop(stack, a);
+           push(stack, at::real(a));
+         },
+         aliasAnalysisFromSchema()),
+     OperatorGenerator(
+         TORCH_SELECTIVE_SCHEMA("prim::imag(Tensor a) -> Tensor"),
+         [](Stack* stack) {
+           at::Tensor a;
+           pop(stack, a);
+           push(stack, at::imag(a));
+         },
+         aliasAnalysisFromSchema()),
```

Comment:
Try removing these and using the ones from `native_functions.yaml` like I suggested above.